static/css/theme-activestate.css

```diff
@@ -83,35 +112,37 @@ ol{
 }
 
 pre {
-    padding: 1rem;
+    padding: 1.25rem;
     margin: 0;
+    margin-top: 1.5rem;
     background: #DAEAFF !important;
     border: 0;
     border-radius: 2px;
-    line-height: 1.15;
+    line-height: 1.5;
 }
 
 pre code {
     color: #2D52BD;
 }
 
 code {
-    background: #E8ECFE;
-    border: 1px solid #E8ECFE;
+    background: #DAEAFF;
+    color: #2D52BD;
+    border: 1px solid #DAEAFF;
```

Comment:
Looks like we use this `DEAAFF` is there an opportunity to move it into a variable like with some of the other colors?